src/main/java/org/folio/oaipmh/helpers/client/ConsortiaClient.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.oaipmh.helpers.client;
+
+import io.vertx.core.json.JsonObject;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.oaipmh.Request;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ConsortiaClient extends InventoryClient {
+
+  private static final Logger logger = LogManager.getLogger(ConsortiaClient.class);
+
+  private static final String USER_TENANTS_ENDPOINT = "/user-tenants";
+
+  public List<String> getUserTenants(Request request) {
+    List<String> result = new ArrayList<>();
+    var endpoint = request.getOkapiUrl() + USER_TENANTS_ENDPOINT;
+    HttpGet httpGet = httpGet(request, endpoint);
```

Comment:
it's better to limit=1, in central tenant we'll have too many records here

Reply:
Added